<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product</class>
 <widget class="QWidget" name="product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>На главную</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teProduct">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>681</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Окно отладки</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>271</width>
     <height>299</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Название отдела</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="leCompanyNameProduct"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Название компании заказчика</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leNameProduct"/>
    </item>
    <item row="9" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnUpdate">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Название проекта</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="leComandNameProduct"/>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="leVersionProduct"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Последняя версия</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSelectAll">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Показать все IT продукты</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twProduct">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IT продукты</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leProductNameSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string extracomment="Введите ФИО Ученика для поиска"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>381</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
